### DISCUSSION TITLE
Proxy Reverse for docker image

### DISCUSSION THREAD

**Discussion by DierlysCSRodrigues** [healthchecks/healthchecks#627] [Q&A] [answered]
Hi

I'm using the docker image to deploy the service and I'm trying to use another container with NGNIX for reverse proxy, however without success.

Do you have any example using NGNIX in docker with reverse proxy?

Or some other alternative ... how could I use UWSGI with traefik for example.. ?

**Comment by cuu508** [comment, marked as answer]
What specific problems are you running into?

One relatively easy way to get nginx + web app set up is to use [dokku](https://dokku.com/).

**Reply by DierlysCSRodrigues** [reply]
Hi, thanks for the reply.

I didn't find any specific problem, I just wanted to know if you had any examples using ngnix.

But, I managed to perform the redirection to HTTPS, using traefik.

Thanks !!!
